files/rconcmds/rcon-cmds-daemon.sh

```diff
@@ -11,11 +12,11 @@
 # shellcheck source=start-utils
 . ${SCRIPTS:-/}start-utils
 
-run_comand(){
-  rcon_cmd=$(echo "$1" | tr -d '"')
-  logAutopause "RCON: running - $rcon_cmd"
+run_command(){
+  rcon_cmd=$1
```

Comment:
I'm surprised bash didn't require this to be quoted as
```suggestion
  rcon_cmd="$1"
```